ldes-server-application/pom.xml

```diff
@@ -130,13 +130,23 @@
             </dependencies>
         </profile>
         <profile>
-            <id>fragmentation-pagination</id>
+            <id>fragmentation</id>
             <dependencies>
+                <dependency>
+                    <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
+                    <artifactId>ldes-fragmentisers-common</artifactId>
```

Comment:
shouldn't this be added to all fragmentation profiles?

Reply:
this actually isn't needed. since there already is a dependency in each module to common